machine-learning/tutorials/SentimentAnalysis.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
@BillWagner are solution files necessary in the repo? When to include them and when they can be omitted?

Reply:
We haven't standardized yet. So, sort of an experiment. 🤷‍♂️